admin/handlers.php

```diff
@@ -175,7 +175,12 @@ function gigpress_prepare_venue_fields() {
 
 
 function gigpress_error_checking($context) {
-	
+
+	/**
+	 * Filter to allow the venue city field to be set as not required.
+	 */
```

Comment:
```suggestion
	/**
	 * Filter to allow the venue city field to be set as not required.
	 *
 	 * @since TBD
 	 *
 	 * @param boolean If the city should be required.
 	 */
```